src/main/java/com/webonise/todoapp/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package com.webonise.todoapp.controller;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.webonise.todoapp.model.User;
+import com.webonise.todoapp.service.UserService;
+
+@RestController
+@CrossOrigin(origins = "*", allowedHeaders = "*")
+
+public class UserController {
+	
+	@Autowired
+	UserService userService;
+	
+	@RequestMapping("/getUser/{username}")
+	public Optional<User> getCredentials(@PathVariable String username) {
```

Comment:
should never get credentials as part of login

Reply:
the backend should verify the credentials supplied by frontend and then issue a bearer token along with an appropriate HTTP code

Reply:
Ok I make changes